### ISSUE TITLE
Please extend the "Browse for Folder" to match path opened

### ISSUE THREAD

**Issue by dylanh724** [notepad-plus-plus/notepad-plus-plus#8513]
## Issue:
The "Browse for Folder" is the lazy browser instead of the full one -- I can't drop a path, so as coders that likely have a deep hierarchy, we have to manually find what we're looking for.

Let's say I have a path, but forgot the file name. Paste the path in >> browse >> it goes straight back to the beginning "Desktop, etc" O_o

## Solution
Either use a full file browser popup, or allow browse to start from the path input instead of top level.

**Comment by sasumner** [comment]
Any particular reason you deleted the new issue template, instead of filling it out?

**Comment by sasumner** [comment]
Presume that the user is speaking of the `...` button functionality in the _Find in Files_ window:

![image](https://user-images.githubusercontent.com/30118311/86462389-8c015e80-bcf9-11ea-8a8e-92ab5942c1c4.png)

**Comment by dylanh724** [comment]
> Any particular reason you deleted the new issue template, instead of filling it out?

Probably the same reason you decided to use the ghetto explorer, bud.

**Comment by sasumner** [comment]
>Probably the same reason you decided to use the ghetto file picker instead of the full one with an address bar and shortcuts

Not sure I see the correlation between those two things.
Also not sure the meaning of "ghetto" used in this context.
But I think I see your point about the _Browse for Folder_ box being not so good.
I would guess it is a feature not used often by users.

>...the template... (Information overload, though : I'd find a way to parse that down.)

The author of Notepad++ is not a native English speaker, and Notepad++ is used around the world and translated into many languages.  For this reason, we often get issues filed that are hard to follow/interpret.
The template is designed (well, one reason for it) to try to step-by-step extract information from a user that might be struggling to express themselves in English.

It also attempts to provide a common basis for the writer and the reader by soliciting steps-to-reproduce, even for English speakers.
For example, my interpretation of your issue was just a guess.  If you'd have said, "Press Shift+Ctrl+f to invoke Find-in-Files, then press the `...` button and notice that the _Browse For Folder_ box..." it would have removed any doubt.

Template also requests version information; that should be the first thing anyone thinks of when submitting an issue.

If you have ideas on streamlining the issue template, we'll certainly take them under consideration.

**Comment by dylanh724** [comment]
@sasumner https://github.com/notepad-plus-plus/notepad-plus-plus/issues/8517

**Comment by mere-human** [comment]
I'd like to work on it.

**Comment by mere-human** [comment]
### Description of the Issue
Browse For Folder doesn't support a file path.

### Steps to Reproduce the Issue

1. Open in Notepad++ Menu > _Search_ > _Find..._
2. Open _Find in Files_ tab
3. Click on `...` button in _Directory_ section
![image](https://user-images.githubusercontent.com/9664141/104060676-ee36a100-51ff-11eb-815f-4e15fa7e6117.png)

### Expected Behavior
File Open dialog appears similar to File > Open that allows path operations.

### Actual Behavior
Browse for Folder dialog doesn't allow editing or pasting the path as text.

![image](https://user-images.githubusercontent.com/9664141/104060627-d7904a00-51ff-11eb-95be-4a6cbb8c0cae.png)

### Debug Information
Notepad++ v7.9.2   (32-bit)
Build time : Dec 31 2020 - 03:58:36
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS Name : Windows 10 Enterprise (64-bit) 
OS Version : 2009
OS Build : 19042.685
Current ANSI codepage : 1252
Plugins : ComparePlugin.dll DSpellCheck.dll HexEditor.dll mimeTools.dll NppConverter.dll NppExport.dll

**Comment by sasumner** [comment]
@mere-human I assigned you.  (Hopefully that doesn't get me in trouble with @donho for doing that) :-)

**Comment by donho** [comment]
@sasumner 
>  (Hopefully that doesn't get me in trouble with @donho for doing that) :-)

You're doing the right thing.

@mere-human 
> Expected Behavior
> File Open dialog appears similar to File > Open that allows path operations.

So it would be a "Open file dialog" but it opens only the folder?

**Comment by sasumner** [comment]
>So it would be a "Open file dialog" but it opens only the folder?

The expectation that the result is somewhat similar to the following:

![image](https://user-images.githubusercontent.com/30118311/104068262-9b92c000-51d2-11eb-93d8-f6107e14e0ea.png)
